readme.md

```diff
@@ -1,60 +1,14 @@
-# analytics-node [![CircleCI](https://circleci.com/gh/segmentio/analytics-node.svg?style=svg&circle-token=68654e8cd0fcd16b1f3ae9943a1d8e20e36ae6c5)](https://circleci.com/gh/segmentio/analytics-node)
```

Comment:
Would it make sense to keep some description of the module such as

```
# analytics
Analytics helps you measure your users, product, and business. It unlocks insights into your app's funnel, core business metrics, and whether you have product-market fit.
```

Reply:
yeah, definitely

Reply:
So this is back again

Reply:
See that the bunch of info you keep in this paragraph has `segment` information. IMO, it would look nicer if we get rid of those as well. So ending up with something like

```
# analytics-node

Analytics helps you measure your users, product, and business. It unlocks insights into your app's funnel, core business metrics, and whether you have product-market fit.
```